android/src/main/java/com/applovin/reactnative/AppLovinMAXNativeAdView.java

```diff
@@ -170,6 +243,45 @@ public void loadAd()
         }
     }
 
+    public void updateAssetView(final int assetViewTag, final String assetViewName)
+    {
+        if ( assetViewName.equals( "TitleView" ) )
```

Comment:
nit: in Java, to avoid NPEs, we operate on the string first. So `"TitleView".equals( assetViewName )`.